src/main/java/org/broadinstitute/hellbender/tools/sv/stratify/SVStratificationEngineArgumentsCollection.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.hellbender.tools.sv.stratify;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.engine.GATKPath;
+import org.broadinstitute.hellbender.utils.tsv.TableUtils;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Arguments for use with {@link SVStatificationEngine}.
+ */
+public class SVStratificationEngineArgumentsCollection implements Serializable {
+    // Command-line arguments
+    public static final String STRATIFY_CONFIG_FILE_LONG_NAME = "stratify-config";
+    public static final String CONTEXT_NAME_FILE_LONG_NAME = "context-name";
+    public static final String CONTEXT_INTERVAL_FILE_LONG_NAME = "context-intervals";
+    public static final String OVERLAP_FRACTION_LONG_NAME = "stratify-overlap-fraction";
+    public static final String NUM_BREAKPOINT_OVERLAPS_LONG_NAME = "stratify-num-breakpoint-overlaps";
+    public static final String NUM_BREAKPOINT_INTERCHROM_OVERLAPS_LONG_NAME = "stratify-num-breakpoint-overlaps-interchromosomal";
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Expected format is tab-delimited and contains columns NAME, SVTYPE, MIN_SIZE, MAX_SIZE, CONTEXT. First line must
+     * be a header with column names. Comment lines starting with {@link TableUtils#COMMENT_PREFIX} are ignored.
+     */
+    @Argument(
+            doc = "Stratification configuration file (.tsv)",
+            fullName = STRATIFY_CONFIG_FILE_LONG_NAME
+    )
+    public GATKPath configFile;
+
+    @Argument(
+            doc = "Context intervals file. Can be specified multiple times.",
```

Comment:
I would call this something more like "repeat track intervals file".  I'm sure you could generalize beyond repeats, but I found this really vague.